### DISCUSSION TITLE
Open AI: Network Problem

### DISCUSSION THREAD

**Discussion by thomas3577-star** [open-webui/open-webui#7910] [Issues]
# Bug Report

## Installation Method

Installed Open WebUI using Docker image.

## Environment

- **Open WebUI Version:** [v0.4.8]
- **Ollama (if applicable):** [v0.5.1]

- **Operating System:** [Windows 11 Enterprise 23H2]
- **Browser (if applicable):** [Chrome 131.0.6778.140, Edge 131.0.2903.99]

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [x] I have included the browser console logs.
- [x] I have included the Docker container logs.
- [x] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

Open WebUI should list all the installed models along with the OpenAI GPT models.

## Actual Behavior:

Open WebUI is listing the installed models and not the OpenAI GPT models.

## Description

**Bug Summary:**
This behavior is showing up after the latest upgrade of Open WebUI to v0.4.8. 

## Reproduction Details

**Steps to Reproduce:**
1. Login to Open WebUI (http://localhost:3000)
2. Click on New Chat 
3. Open the drop-down of Models to choose, no OpenAI GPT models displayed.
4. Go to Admin Panel > Settings > Connections > Open API
5. Click on Configure icon for "https://api.openai.com/v1"
6. Click on "Verify Connection"
7. Error Pop up "Open AI: Network Problem" shows up

## Logs and Screenshots

**Browser Console Logs:**
[Chrome-console-localhost-1734406668063.log](https://github.com/user-attachments/files/18159731/Chrome-console-localhost-1734406668063.log)
[Chrome-HAR.txt](https://github.com/user-attachments/files/18159732/Chrome-HAR.txt)

**Docker Container Logs:**
[Docker-2024-12-17 091014.txt](https://github.com/user-attachments/files/18159792/Docker-2024-12-17.091014.txt)

**Screenshots/Screen Recordings (if applicable):**
![image](https://github.com/user-attachments/assets/37231ff8-48a6-4183-bd54-4918dd91fb46)

**Comment by derek-assurity** [comment]
That's an issue with your Docker config not with Open Web UI. If you're running behind a Proxy/Firewall then Docker will need to be configured to use it.

**Reply by thomas3577-star** [reply]
Thanks @derek-assurity for pointing out the issue. 
I have tried the proxy settings at the Docker Desktop application level, it is still not helping. 
Can you suggest any other workarounds?

**Reply by derek-assurity** [reply]
@thomas3577-star Just ran a quick test and it looks like the error message is misleading. Check the docker logs after testing the OpenAI connection and you should be able to see the cause of the issue. You can check them in the Docker dashboard or run `docker logs open-webui` (assuming that is the name you've given the running image).

**Reply by thomas3577-star** [reply]
Thanks @derek-assurity once again for the timely direction. I did go through the logs and found that the HTTPS_PROXY setting was not working. Referred to the company proxy settings, made the change.

In summary, the fix I applied was including proxy settings as part of Docker command.

docker run --detach --publish 3000:8080 --add-host=host.docker.internal:host-gateway --volume open-webui:/app/backend/data --name open-webui --restart always **--env HTTP_PROXY=Enter Proxy Url --env HTTPS_PROXY=Enter Proxy Url --env NO_PROXY=localhost,127.0.0.1,host.docker.internal** ghcr.io/open-webui/open-webui:main

Now, I can see the Open AI models being listed.
